<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SupplierLabelManager</class>
 <widget class="QDialog" name="SupplierLabelManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Versandscheinmanager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_main">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_center">
     <item>
      <widget class="QTableWidget" name="tableDefinitions">
       <property name="editTriggers">
        <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>50</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>135</number>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Logistikunternehmen</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Labeltyp</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Keywords</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelCarrier">
         <property name="text">
          <string>Logistikunternehmen:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboCarrier"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelLabelType">
         <property name="text">
          <string>Labeltyp:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="editLabelType"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelKeywords">
         <property name="text">
          <string>Keywords (kommagetrennt):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="editKeywords"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_buttons">
     <item>
      <widget class="QPushButton" name="btnNew">
       <property name="text">
        <string>Neu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Speichern</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="text">
        <string>Löschen</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Schließen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
